lib/plugins/aws/lib/naming.js

```diff
@@ -519,7 +519,7 @@ module.exports = {
       functionName
     )}LambdaPermissionCognitoUserPool${this.normalizeNameToAlphaNumericOnly(
       poolId
-    )}TriggerSource${triggerSource}`;
+    )}TriggerSource${triggerSource}`.replace(/[\W_]+/g, '');
```

Comment:
I think we could just wrap the `triggerSource` with `normalizeNameToAlphaNumericOnly`, util that we already have implemented and available in this module, what do you think?

Reply:
Yes, that's neater. I didn't notice that function earlier. Will change it and utilize this function. Thanks for your feedback.